src/runtime/evaluator/eval_stmt.rs

```diff
@@ -40,6 +47,26 @@ impl Evaluator {
         // self.env.insert(assn.dest.clone(), assn.src.clone());
         Ok(())
     }
+    pub fn eval_insert(&mut self, insert: &mut Insert) -> Result<(), String> {
+
+        let row_data = insert.row.val.iter().map(|entry| Entry {
+            name: entry.name.clone(), val: entry.val.clone(),
+        }).collect();
+
+        let row = Row { val: row_data };
+        
```

Comment:
I wont comment everywhere but you can generally look fix the style incosistencies